kubernetes/jobs/create-database.yml.j2

```diff
@@ -18,15 +18,17 @@ spec:
           secret:
             secretName:  db-secrets
             defaultMode: 0600
+        {% if django_admin_fixtures %}
```

Comment:
Please rename it to `initialize_django_admin`.

Reply:
Done.